PURL-TYPES.rst

```diff
@@ -297,6 +297,27 @@ github
       pkg:github/package-url/purl-spec@244fd47e07d1004
       pkg:github/package-url/purl-spec@244fd47e07d1004#everybody/loves/dogs
 
+go
+------
```

Comment:
```suggestion
-------------
```


----

⬆️ this is a demo how a change suggesttion looks like; it is not meant to be applied
the exact "code" i used: 
````markdown
```suggestion
-------------
```
````